### PR TITLE
Law of Common Fate + Law of Closure + Interlinked Intricate Network

### PR DESCRIPTION
**Pull request by jakobvk**
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

### CODE REVIEW

File: form/2d/p5js/closethetriangle/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  noStroke ();
+  fill (150,75,90,100);
+  triangle (150,100,100,300,380,250);
+}
+
+function draw() {
+  
+}
+function mousePressed () {
+  save ('image.png');
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.js]: Please remove the save function for the website

File: form/2d/p5js/closethetriangle/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: [Jakob von Kietzell]
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: no `[]` here

File: form/2d/p5js/closethetriangle/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+author-url: [https://incom.org/profil/7939]
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: no `[]` here

File: form/2d/p5js/closethetriangle/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+title: [closethetriangle]
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: no `[]` here

File: form/2d/p5js/closethetriangle/index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+tags: [P5.js,[FORM], [2D], Law of Closure]
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: no inner `[]` here the only `[]` are the outer ones. Tags like "form" or "2d" are written in lowercase. Laws are written like titles.

File: form/2d/p5js/closethetriangle/index.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+permalink: /[FORM]/[closethetriangle]/
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: no `[]` here. form is written lowercase.  See the example https://raw.githubusercontent.com/fabianmoronzirfas/gestalten-in-code/master/grid/p5js/lines/index.md

File: form/2d/p5js/closethetriangle/index.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+summary: [can you see the triangle?]
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: no [] here

File: motion/p5js/commonfateplanets/index.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.md]: see the remarks for the first index.md

File: form/2d/p5js/closethetriangle/index.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
+function setup() {
+  createCanvas (500,500);
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.js]: the canvas needs to be assigned to the sketch id div. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code

File: form/2d/p5js/closethetriangle/index.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+  background (150,75,90,100);
+  
+  
+  
+  fill (20);
+  rect (0,0,width/2,height);
+  
+  noStroke ();
+  fill (150,75,90,100);
+  triangle (150,100,100,300,380,250);
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.js]: Very magic numbers.

File: form/2d/p5js/closethetriangle/index.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jakob von Kietzell
+author-url: https://incom.org/profil/7939
+title: closethetriangle
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: Make it a real title please: "Close the Triangle"

File: form/2d/p5js/closethetriangle/index.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+tags: [P5.js,form, 2d, Law of Closure]
+permalink: /form/closethetriangle
+summary: can you see the triangle?
```
Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.md]: This is very short. Please explan a bit further.

File: motion/p5js/commonfateplanets/index.js [hunk 13]
```diff
@@ -0,0 +1,148 @@
+// Kreis1
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.js]: English please.

File: motion/p5js/commonfateplanets/index.js [hunk 14]
```diff
@@ -0,0 +1,148 @@
...
+var circleX4;
+var circleY4;
+
+//Kreis5
+var circleX5
+var circleY5
+
+
+function setup() {
+  createCanvas (500,500);
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.js]: the canvas needs to assigned to the sketch id div. see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code

File: motion/p5js/commonfateplanets/index.js [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+  circleY4 =250;
+  circleX5 =-60;
+  circleY5 =560;
+}
+
+function draw() {
+  background (255);
+  noStroke ();
+
+//Kreis1
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.js]: english

File: motion/p5js/commonfateplanets/index.js [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+  circleX = circleX+1;
+  circleY = circleY+1;
+  fill (120, 30, 100,50);
+  ellipse (circleX, circleY, sizeX,sizeY);
+
+//Kreis2
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.js]: english

File: motion/p5js/commonfateplanets/index.js [hunk 17]
```diff
@@ -0,0 +1,148 @@
...
+  fill (120, 30, 100,50);
+  ellipse (circleX, circleY, sizeX,sizeY);
+
+//Kreis2  
+  circleX2 = circleX2+1;
+  circleY2 = circleY2+1;
+  fill (120, 30, 100,50);
+  ellipse (circleX2, circleY2, sizeX2,sizeY2);
+
+//Kreis3
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.js]: …

File: motion/p5js/commonfateplanets/index.js [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+
+//Kreis5
+if (circleX5 >= width+60) {
+  circleX5 =-60;
+  circleY5 =560;
+}
+  
+}
+
+function mousePressed () {
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.js]: please remove the image saving code.

File: motion/p5js/commonfateplanets/index.md [hunk 19]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jakob von Kietzell
+author-url: https://incom.org/profil/7939
+title: commonfateplanets
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.md]: Title please.

File: motion/p5js/commonfateplanets/index.md [hunk 20]
```diff
@@ -0,0 +1,9 @@
...
+tags: [P5.js,motion, commonfate, Law of Common Fate]
+permalink: /motion/commonfateplanets/
+summary: some planets moving around. two belong together.
```
Comment (fabianmoronzirfas) [inline motion/p5js/commonfateplanets/index.md]: This is also very short.

File: projects/interlinked_intricate_network/index.md [hunk 21]
```diff
@@ -0,0 +1,218 @@
...
+authors: 
+  - "Jakob von Kietzell"
+authors-url: 
+  - "https://github.com/jakobvk"
+layout: project
+title: Interlinked Intricate Network
+type: project
+---
+
+# Interlinked Intricate Network
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: Please start at h2 the h2 will be added by the system.  See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax

File: projects/interlinked_intricate_network/index.md [hunk 22]
```diff
@@ -0,0 +1,218 @@
...
+![](./assets/images/009_splash.png)
+
+
+## Analog Algorithms
+ 
+For this project an algorithm was developed, that can be executed by humans – in other words: analog. The function pattern was repeatedly evaluated and optimized iteratively.
+ 
+The project was developed for the class “Gestalten in Code – Die Grundlagen Generativer Gestaltung” by Fabian Morón Zirfas at the University of Applied Sciences Potsdam, Faculty of Design.
+
+<hr />
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: hr in markdown is 

    ---

with a breakline before and after it.

File: projects/interlinked_intricate_network/index.md [hunk 23]
```diff
@@ -0,0 +1,218 @@
...
+## First Algorithm
+
+The first algorithm was only allowed to be up to 5 sentences long and created in 5 minutes. 
+These were the results. 
+
+>Draw a circle in the center of your sheet of paper. This circle is surrounded in same distances by further circles that are evenly subdivided into pieces. There’s no gap at the same line in the circle that is further away from the centre. 
+ 
+![](./assets/images/001_splash.png)
+
+### evaluation
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: Title Case

File: projects/interlinked_intricate_network/index.md [hunk 24]
```diff
@@ -0,0 +1,218 @@
...
+
+The first algorithm was very open and a little hard to read because of the messy handwriting in a hurry. The results didn’t meet my expectations at all and were very different to one another.
+
+<hr />
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: same here ---

File: projects/interlinked_intricate_network/index.md [hunk 25]
```diff
@@ -0,0 +1,218 @@
...
+
+## New Algorithm
+
+### Inspiration
+
+I decided to start all over.
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: Loose the `I` - write from an objective perspective.

File: projects/interlinked_intricate_network/index.md [hunk 26]
```diff
@@ -0,0 +1,218 @@
...
+
+## New Algorithm
+
+### Inspiration
+
+I decided to start all over. 
+The new thought was to have two separated areas on one work surface. In these areas there are three mushroom-like, recursive, smaller and smaller objects. There is an undefined open shape crossed by lines that are defined by objects, so that the two areas are connected.<br />
+The source for this thought was a conversation about fractals. The output of the algorithm should be some object that consists of lots of smaller copies of itself. 
+
+*Definition Fractals*
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: H4 not bold?

File: projects/interlinked_intricate_network/index.md [hunk 27]
```diff
@@ -0,0 +1,218 @@
...
+A fractal is a never-ending pattern. Fractals are infinitely complex patterns that are self-similar across different scales. They are created by repeating a simple process over and over in an ongoing feedback loop. Driven by recursion, fractals are images of dynamic systems – the pictures of Chaos. Geometrically, they exist in between our familiar dimensions. Fractal patterns are extremely familiar, since nature is full of fractals. For instance: trees, rivers, coastlines, mountains, clouds, seashells, hurricanes, etc. Abstract fractals – such as the Mandelbrot Set – can be generated by a computer calculating a simple equation over and over.
+
+![example_fractals](http://fractalfoundation.wolfesongs.com/wp-content/uploads/2009/01/sierpinski-zoom41.gif)
+
+[Source: Website Fractal Foundation](http://fractalfoundation.org/resources/what-are-fractals/)
+ 
+
+### execution
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: Title case

File: projects/interlinked_intricate_network/index.md [hunk 28]
```diff
@@ -0,0 +1,218 @@
...
+>9. Repeat this for all the new triangles until the new triangles are smaller than the stroke width of your text/ paint tool.
+>10. Read the following step completely before executing
+>11. Draw straight lines with text/ paint tool no 2., that are each on the one hand side connected with the corners of the triangles drawn in step 3 and on the other hand side with the free corner of the triangle drawn in step 3 facing the other one.
+>12. Continue drawing the lines until you reach the edge of the work surface.
+
+![](./assets/images/002_normal.png)
+ 
+### constant parameters
+ 
+One work surface<br />
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: if you want a breakline in markdown just add to whitespaces at the end of the line

File: projects/interlinked_intricate_network/index.md [hunk 29]
```diff
@@ -0,0 +1,218 @@
...
+In this iteration the algorithm got rid of the three areas and focussed on the recursive triangles. 
+
+#### What went wrong? What worked out?
+ 
+The algorithm was still long and too complicated to execute. It took people taking part long and they didn’t enjoy. <br />
+Although it took people long, every output is comparable and I already really liked the looks. 
+ 
+  <hr />
+  
+## 2nd iteration
```
Comment (fabianmoronzirfas) [inline projects/interlinked_intricate_network/index.md]: Title case

File: form/2d/p5js/closethetriangle/index.js (hunk 10 above)
Comment (jakobvk) [inline form/2d/p5js/closethetriangle/index.js]: hey fabian, in this sketch it doesn't make sense for me to exchange the magic numbers as its just a picture anyways so i will leave it like this

Comment (fabianmoronzirfas) [inline form/2d/p5js/closethetriangle/index.js]: actually it makes sense (or I would not ask for it). If somebody changes the size of the sketch your picture will not look the same

**Comment by fabianmoronzirfas** [comment]
This image has not the right size https://github.com/fabianmoronzirfas/gestalten-in-code/pull/24/files#diff-40700856a91ac6004411f2fba7a8c8de

**Pull request merged**